docs/tutorials/google/calibration_faq.md

```diff
@@ -0,0 +1,311 @@
+
+Alternatively, one can run a Loschmidt echo with random circuits using Floquet calibration, and see that the overall return 
+probability is higher than before. Even though the Loschmidt Echo random circuit may look very different from the actual circuit
+you are running, the gate compensations should result in a decent improvement in the return probabilities.
+
+See [this tutorial for an example of using Loschmidt echo](./echoes.ipynb). Here is an example of applying Floquet and XEB
+corrections to a 16-site Loschmidt echo circuit (higher is better):
+
+![Calibration benchmark with Loscmidt echoes](./loschmidt_echo_with_calibration_example_results.png)
+
+Here is another example of applying calibrations to a linear chain circuit that contains only Z gates and `cirq.SQRT_ISWAP` gates
+(lower is better):
+
+![Calibration benchmark with linear chain circuit](./linear_chain_with_calibration_example_results.png)
+
+## How should I test Floquet / XEB Calibration on my circuit?
+
+* First, you should have a circuit whose performance needs to be improved.  Defining a metric of success is important to 
+  clarify whether the procedure was successful. (E.g., for the case of the Fermi-Hubbard experiment, calculating the 
+  spin-charge density of each site).
+ 
+* Make sure you get a reservation budget and then 
+  [reserve a block of time using the cloud console](https://console.cloud.google.com/quantum/processors/) to test various
+  permutations of your circuit. Expect that each calibration would take about 1.6s per qubit and 28s per moment of additional 
+  time to your regular circuit runs.
+ 
+* Run the circuit against both a perfect simulator and a coherent noise simulator to get valid baselines for performance.
+
+* Using [this tutorial for Floquet calibration](./floquet_calibration_example.ipynb) as a guide, introduce Floquet characterization 
+  and resulting compensating Z phase gates into your circuit, and compare the results with Floquet calibration turned on and turned off.
+
+In order to ensure an apt comparison, it is important to keep the following points in mind:
+
+* Make sure you pick your qubits wisely. 
+  - Use parallel readout heatmap to avoid any obviously broken qubits.
+  - Use two-qubit error heatmaps to choose various patches of qubits with good two-qubit gate performance.
+  - Use a mini-benchmark circuit to sanity check both qubit choice and basic circuit design.
+  - If in doubt, use parallel XEB to check for issues like drift / TLS.
+
+* Start small with a low number of qubits (e.g. 2 qubits, with 5-10 layers of 2 qubit gates), and increase the number of qubits 
+  and gate depth.
+* Do not increase the gate depth past the T1 time of the qubits.
+* Use out of the box re-compilation first. If that doesn’t work, then try custom recompilation.
+
+For a complete set of best practices, check the following [guide](../../google/best_practices.md).
+
+## How to compensate for the parasitic c-phase φ angle?
```

Comment:
After reading this all, maybe this should be added to the end of "## What if the compensation based on single-qubit phases is not enough?", WDYT?

Reply:
I think linking to best practices here makes sense because it's describing many of the practices in that guide. But I agree it also makes sense to also add a note to the end of the "## What if... is not enough" section  - so I kept this note and added a similar note to that section.

Reply:
Have you committed the changes to the other section? I don't see them there?

Reply:
Yes! Sorry, I am referring to [these lines](https://github.com/quantumlib/Cirq/pull/4442/commits/bd4461d3f783b7ec414d6c97d331f922016a0ab1#diff-30d6b9220224c9f0f4318ab716cc655106caf0255b7996e0899c2b53d29009a5R219-R221) from bd4461d3f783b7ec414d6c97d331f922016a0ab1:

```markdown
If these strategies are still not enough, make sure you are following all best practices outlined in this 
[guide](../../google/best_practices.md), for example avoiding qubits with high errors.
```

Reply:
We might have misunderstood each other. I was thinking about referencing "## How to compensate for the parasitic c-phase φ angle?" from "## What if... is not enough" because those two sections are very similar to each other and they related to similar issues.

I'm not sure if linking best practices makes much sense in "## What if... is not enough". This section describes how to correct the gates, which is not a topic of best practices document.

Reply:
Ah, I see, thanks. I updated the text and link to go to the parasitic c-phase angle section, PTAL.

Reply:
Yes, this looks good I think!